source/agora/stats/Stats.d

```diff
@@ -133,6 +133,12 @@ public struct Stats (ValueType, LabelType)
         }
         return res;
     }
+
+    /// Clears stats
+    public void clearStats() nothrow @safe
```

Comment:
Style : Space before parenthesis

Reply:
Also since this is a stats only structure, maybe the "stats" part is redundant ?